ui/src/components/MobileClientListViewItem.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import { Alert, ListView, Row, Col, DropdownKebab, MenuItem} from 'patternfly-react';
+
+import MobileClientServiceChart from './MobileClientServiceChart';
+import ComponentSectionLabel from './common/ComponentSectionLabel';
+import MobileClientBuildList from './MobileClientBuildList';
+
+import './OverviewListItemHeader.css';
+
+// todo 
+const mobileServices = {
+    "mobileServices": {
+        "bound": 1,
```

Comment:
What mean this numbers 1 and 2? Could be they in a const with a name in order to make clear their propose?

Reply:
Its just dummy data for now to populate the patternfly donut chart. This will be solved later on.

Reply:
👍